### PR TITLE
chore(readme): add nodejs logo

### PR DESCRIPTION
**Pull request by AugustinMauroy**
Modification of the `readme.md` to add the nodejs logo instead of a simple link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# [nodejs.org](https://nodejs.org/)
+<p align="center">
+  <br />
+  <a href="https://nodejs.dev">
+    <img src="./public/static/images/logo-light.svg" width="200"/>
```
Comment (ovflowd) [inline README.md:4]: This link is broken.

**Comment by ovflowd** [comment]
I think, overall, this README needs some heavy modifications. Feel free to make it look alike the one from nodejs.dev

**Comment by AugustinMauroy** [comment]
I wasn't sure how far I could go. I'm going to redo everything from scratch and get back to you

**Comment by AugustinMauroy** [comment]
The title of pr have to change but IDK. What I have to put.

**Comment by AugustinMauroy** [comment]
@ovflowd I will close this pr you do the job.

**Pull request closed without merging**